src/clients/SpokePoolClient.ts

```diff
@@ -260,6 +260,11 @@ export class IndexedSpokePoolClient extends clients.SpokePoolClient {
   }
 
   protected async _update(eventsToQuery: string[]): Promise<clients.SpokePoolUpdate> {
+
+    if (this.pendingBlockNumber === this.deploymentBlock) {
+      return { success: false, reason: clients.UpdateFailureReason.AlreadyUpdated }; // @todo: Update reason
```

Comment:
To be changed once https://github.com/across-protocol/sdk/pull/729 is merged.

```suggestion
      return { success: false, reason: clients.UpdateFailureReason.NotReady };
```

Reply:
Done: https://github.com/across-protocol/relayer/pull/1781/commits/491e94787f4f3ebbe60cf950dc0ff87c27841f2b